marketplace/mainTemplate-byol_2019.json

```diff
@@ -291,7 +291,7 @@
               "publisher": "couchbase",
               "offer": "couchbase-server-enterprise",
               "sku": "[parameters('license')]",
-              "version": "3.0.2"
+              "version": "18.4.0"
```

Comment:
I was expecting this to be 6.6.0 or some kind of Couchbase version, but I'm assuming the previous versioning was wrong here, as it is now being used to represent the base vm version.

Reply:
Yeah.. This is the base VM image version, not the couchbase version